deploy.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: rbac-frontend
+objects:
+  - apiVersion: cloud.redhat.com/v1alpha1
+    kind: Frontend
+    metadata:
+      name: rbac
+    spec:
+      envName: ${ENV_NAME}
+      title: RBAC
+      deploymentRepo: https://github.com/RedHatInsights/insights-rbac-ui
+      API:
+        versions:
+          - v1
+      frontend:
+        paths:
+          - /apps/rbac
+      image: ${IMAGE}:${IMAGE_TAG}
+      navItems:
+        - appId: 'rbac'
+          title: 'My User Access'
+          href: '/settings/my-user-access'
+        - title: 'User Access'
+          expandable: true
+          permissions:
+            - method: 'isEntitled'
+              args: ''
+          routes:
+            - appId: 'rbac'
+              title: 'Users'
+              href: '/settings/rbac/users'
+            - appId: 'rbac'
+              title: 'Roles'
+              href: '/settings/rbac/roles'
+            - appId: 'rbac'
+              title: 'Groups'
+              href: '/settings/rbac/groups'
+            - appId: 'rbac'
+              title: 'Red Hat Access Requests'
+              href: '/settings/rbac/access-requests'
+              notifier: 'chrome.accessRequests.hasUnseen'
```

Comment:
This is the first I've seen of a `notifier`. We don't support it in the operator yet. If it's just a string I can do that tomorrow @Hyperkid123  @karelhala

Reply:
@BlakeHolifield  Yeah it is just a string. It is used to access certain properties in the chrome store (internal data). In this case, it will show a little bell icon in the link if the object property described by the string is truthy.